### ISSUE TITLE
Wrong command line in "Attaching after JVM startup"

### ISSUE THREAD

**Issue by WolfgangHG** [jenkinsci/lib-file-leak-detector#153]
### Describe your use-case which is not covered by existing documentation.

The chapter [Attaching after JVM startup](https://github.com/jenkinsci/lib-file-leak-detector#attaching-after-jvm-startup) uses this command:
`java -jar path/to/file-leak-detector.jar 1500 threshold=200,strong`

This does not work for me: "file-leak-detector-1-16.jar" does not define a Main class. And when declaring the class to invoke, there is an error about missing classes ("ags4j" stuff).

You have to use the file "file-leak-detector-1.16-jar-with-dependencies.jar"

**Comment by basil** [comment]
So you would like the documentation to be updated to read `file-leak-detector-1.16-jar-with-dependencies.jar` rather than `file-leak-detector-1-16.jar`?

**Comment by WolfgangHG** [comment]
@basil Exactly. At least this worked for me. 
I don't know whether the snippets in the chapter "Usage" are also affected.

**Comment by basil** [comment]
> Exactly.

And you decided to file an issue for this rather than a pull request?

**Comment by WolfgangHG** [comment]
Yes - I considered it dangerous to send pull request after having used this project for the first time and not having any detailed knowledge ;-). If you prefer it, I can create a pull request, but as I wrote before, I don't know whether the chapter "Usage" would also have to be changed.

**Comment by basil** [comment]
Yes, please submit a pull request. Don't worry about that other chapter... you've already demonstrated that your proposed improvement is a strict improvement over the status quo.
